### ISSUE TITLE
The dashboard for 1.1.0 release discussions

### ISSUE THREAD

**Issue by jbkoh** [BrickSchema/Brick#52]
I will use this thread as a dashboard for the next release discussions. Please comment below if I missed anything. This discussion is based on the current [dev repo](https://github.com/BrickSchema/brick-owl-dl). Once most of the items below reach some agreement, we will try to reach out to other people. Then, we will pre-release the version (guess it'd be 1.1.0).

# Technical Reviews
- [x] Review/discuss if the concept of maintaining TagSets' associated Tags is feasible.
    - This is one of the major concepts we are introducing in the new version. With this, we can formally define classes (other than just interpret a TagSet with its underscore-separated Tags.)
    - If anybody is interested in the details, we can have a call to review the design. (And I'd appreciate it.)
- [x] Review the correctness of the current hierarchy and the Tags<->Class (i.e., TagSets) relationships.
    - Please visit our new website (thanks, @shreyasnagare!). You can navigate the class hierarchy better: https://brickschema.org/ontology/1.1.0/#Classes
    - This may be done in several subthreads.
- [x] ~~Grouping points by FunctionalBlocks~~
    - I will publish this issue later tomorrow based on the discussion between me and Aslak.
- [x] Is ``feeds`` transitive? 
    - https://github.com/BrickSchema/brick-owl-dl/issues/15
- [x] ~~Introduce Lighting System~~
    - https://github.com/BrickSchema/Brick/issues/29
    - Independent to the above issue, we have an external group interested in this. However, as the contribution has been old, I would like to finalize and merge into the next release.
- [x] ~~Introduce Electrical Domain~~
    - https://github.com/BrickSchema/Brick/issues/28
    - https://groups.google.com/forum/#!topic/brickschema/6QvW5kJEmks
    - Similar to Lighting System
- Dangling issues
    - [x] https://github.com/BrickSchema/Brick/issues/49
    - [x] https://github.com/BrickSchema/Brick/issues/45
- Other possible features
    - [ ] Unit: We had a long discussion of this before, but we can just settle with simple unified codes.

# Non-technical Reviews

- [x] Replacing the terminology "TagSet" with "Class"
    - I won't create a separate thread for this but if there is anybody who is not sure
- [x] Is ``Quantity`` generic enough to express what we want to model? I feel ``Property`` more fitting, but it is confusing with RDF/OWL's general Property concept. I can live with ``Quantity`` but want to learn others' opinions.

# Remaining Procedure
1. Once this review is done, we will publish a beta version and get reviews publicly. 
2. Once the public review/revision is done, we will publish the next version. (Either v1.1.0 or v2.0.0)

This is a reduced version of my original proposal in the other thread.

**Comment by jbkoh** [comment]
Electrical_System may require a controversial concept, so I'd like to defer it for next versions.

For Lighting_System, we already included many of the discussed concepts in the past. However, I would like to correct one concept as in #77 . Other than this, we need more discussions and inputs from the experts. We'd better coordinate with ANSI C137 for more precise and complex modeling.

**Comment by gtfierro** [comment]
@jbkoh I'm closing this issue because we've addressed all of the items raised. Continuing discussion of the final v1.1 items in https://github.com/BrickSchema/Brick/pull/93
